Exercise_B/index.html

```diff
@@ -19,16 +25,81 @@
 
   <body>
     <!--Header (flower shop logo block element)-->
+    <header>
+      <div class="logo container">
+        <img src="assets/images/logo.png" alt="Logo of the shop">
+      </div>
+    </header>
     <!--Banner Jumbotron of the image-->
+  
+    <div class="jumbotron" style="background-image:
+     linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), 
+    url(./assets/images/flowershop.jpg)">
+      <div class="d-flex justify-content-center align-content-center">
+      <h1>Beautiful Flowers</h1>
+      </div>
+      <div class="d-flex justify-content-center align-content-center">
+      <a class="btn btn-success btn-lg" href="#" role="button" id="learnMore">Learn more</a>
+      </div>
+    </div>
+
+<!--Making the footer with bootstrap grid-->
+
+<div class="container-fluid" id="footerContainer">
+
+<div class="row">
+
+      <div class="col-xs-12 col-sm-12 col-md-6 col-xl-3" id="footerSelector">
+        <img src="assets/images/flower-icon.png">
+        <p>Flowers for every kind of situation</p>
+      </div>
+
+
+
+      <div class="col-xs-12 col-sm-12 col-md-6 col-xl-3" id="footerSelector">

```

Comment:
Great use of the bootstrap column system to specify how many elements you want depending on the windows size. Super work!